### PR TITLE
Timeline component 

### PR DESCRIPTION
**Pull request by gustavoquinalha**
Reference #33

### CODE REVIEW

File: timeline/component.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name":"timeline",
+  "author": "gustavoquinalha",
+  "dependencies": []
```
Comment (fabiorogeriosj) [inline timeline/component.json]: The `timeline ` have dependenies with `base` and `grid` component?

File: timeline/style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  color: #bdbbbb;
+  border: 1px solid #d7d2d2;
+  font-size: 12px;
+}
+.user-photo-1 {
```
Comment (fabiorogeriosj) [inline timeline/style.css]: I think it is better if add classe generic, example: `.avatar-group`.

File: timeline/style.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  left: 16px;
+}
+.space-marker {
+  z-index: 9
+}
+.marker {
+  margin: 0 auto;
+  margin-top: 10px;
+}
+.calendar-date {
```
Comment (fabiorogeriosj) [inline timeline/style.css]: It's best to use `text-huge` in the base component.

File: timeline/style.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
+.timeline-line:after {
```
Comment (fabiorogeriosj) [inline timeline/style.css]: Rename component just `.timeline`

File: timeline/style.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  content: "";
+  position: absolute;
+  width: 2px;
+  height: 100%;
+  background: #dddddd;
+  z-index: 1;
+  left: 16px;
+}
+.space-marker {
```
Comment (fabiorogeriosj) [inline timeline/style.css]: Is it possible to just use a div with the `.marker` class and add the other details with`:after` and / or `:before`?

File: timeline/style.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  min-height: 120px;
+}
+.calendar-date span {
+  font-size: 4em
+}
+.group-user-photo {
+  -webkit-transform: translateX(10px);
+          transform: translateX(10px);
+}
+.user-photo {
```
Comment (fabiorogeriosj) [inline timeline/style.css]: Since this feature is only for the `timeline`, start the class with `.timeline .user-photo`

**Review by fabiorogeriosj — COMMENTED** [review]
I've pointed out some items that can be improved.

File: timeline/style.css (hunk 3 above)
Comment (gustavoquinalha) [reply timeline/style.css]: and the size? can i put inline?

File: timeline/style.css (hunk 5 above)
Comment (gustavoquinalha) [reply timeline/style.css]: .timeline-group .row:last-child .timeline {
  display: none
}
can i use this?

Comment (fabiorogeriosj) [reply timeline/style.css]: I dont understand about `timeline-group`, is it possible to work without it?

File: timeline/style.css (hunk 3 above)
Comment (fabiorogeriosj) [reply timeline/style.css]: for example I think the huge size is ok!

**Pull request merged**